Resources/doc/reference/form_types.rst

```diff
@@ -370,6 +370,9 @@ According the choice made only associated fields are displayed. The others field
 map
   Associative array. Describes the fields that are displayed for each choice.
 
+**TIP**: To mask a subfield replace `.` by `__`.
```

Comment:
Everything is a BC-break, you have to make some calls

![everything is a BC-break](https://imgs.xkcd.com/comics/workflow.png)